tests/main/parallel-install-services/task.yaml

```diff
@@ -0,0 +1,47 @@
+summary: Checks for parallel installation of sideloaded snaps containing services
+
+prepare:
+    snap set system experimental.parallel-instances=true
+
+restore:
```

Comment:
Same request as in the other PR, please switch this to `restore: |` for preventing simple mistakes in case someone tweaks this test later.

Reply:
I'll update the other PRs too